nbgrader/apps/feedbackapp.py

```diff
@@ -37,38 +26,74 @@ class FeedbackApp(BaseNbConvertApp):
 
     examples = Unicode(dedent(
```

Comment:
Also, maybe dumb question, but what are the use of making this traitlets if it's not configurable ? I'm not sure where it is used...